src/main/java/javax/security/enterprise/identitystore/IdentityStore.java

```diff
@@ -67,7 +67,7 @@
  */
 public interface IdentityStore {
     
-    public static final Set<ValidationType> DEFAULT_VALIDATION_TYPES = EnumSet.of(VALIDATE, PROVIDE_GROUPS);
+    Set<ValidationType> DEFAULT_VALIDATION_TYPES = EnumSet.of(VALIDATE, PROVIDE_GROUPS);
```

Comment:
Every field declaration in the body of an interface is implicitly public, static, and final. It is permitted to redundantly specify any or all of these modifiers for such fields. 
Defined in the Java Language Spec (http://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.3)